Source/WebCore/Modules/encryptedmedia/MediaKeySystemController.cpp

```diff
@@ -65,7 +65,7 @@ void provideMediaKeySystemTo(Page& page, MediaKeySystemClient& client)
 void MediaKeySystemController::logRequestMediaKeySystemDenial(Document& document)
 {
     if (RefPtr window = document.domWindow())
-        window->printErrorMessage(makeString("Not allowed to access MediaKeySystem."));
```

Comment:
Is there a way to change the definition of `makeString` to disallow inputting a single argument to encourage the use of `_str`?

Reply:
Maybe, I need to look into it.